libs/orchestrator-ui-components/src/lib/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { EuiBreadcrumb, EuiBreadcrumbs, EuiSpacer } from '@elastic/eui';
+import { useRouter } from 'next/router';
+import { containsUuid4, isUuid4, upperCaseFirstChar } from '../../utils';
+
+export const Breadcrumbs = () => {
+    const router = useRouter();
+    // Setup initial breadcrumbs with navigation to home
+    const breadcrumbs: EuiBreadcrumb[] = [
+        {
+            text: 'Start',
+            href: '/',
+            onClick: (e) => {
+                e.preventDefault();
+                router.push('/');
+            },
+        },
+    ];
+
+    // Handle remaining breadcrumbs dynamic
+    const parts = router.asPath.split('/');
+    parts.forEach((p, index) => {
+        if (index > 0) {
+            let link = parts.slice(0, index + 1).join('/');
+            if (link.includes('?')) {
+                link = link.split('?').slice(0, -1).join();
+            }
+            // Handle UUID's: so you can have breadcrumb like: `Start / Subscriptions / 12312aa-cbc ...`
+            let text = isUuid4(p) ? p : upperCaseFirstChar(p);
+            if (text.includes('?')) {
+                text = upperCaseFirstChar(text.split('?').slice(0, -1).join());
+            }
+
+            breadcrumbs.push({
+                text: text,
+                href: link,
+                onClick: (e) => {
+                    e.preventDefault();
+                    if (
+                        text === 'Subscriptions' &&
+                        containsUuid4(router.asPath)
+                    ) {
+                        // Todo: make URLS more consistent or design a better way to handle breadcrumbs
+                        // When possible try to use the browser back; so the user has active tab + correct page
+                        router.back();
```

Comment:
This one back-fires (no pun intended) when you open a specific subscription detail in a new tab: http://localhost:4200/subscriptions/9f42c702-bed7-4c27-b8e8-05d446053065.

When you click on subscriptions in the breadcrumb you will end up in the browsers empty new-tab page.

Reply:
Removed this and floowed @wouter1975 specs 100% : no smart routing in the BreadCrumbs: only links to top level pages; So filter, tab and paginate will reset to defaults